man/fi_av.3.md

```diff
@@ -436,6 +436,11 @@ fi_av_set_user_id.
 
   See the user ID section below.
 
+- *FI_NO_CONNECT*
+: This flag indicates that AV insertion operations should fail if there
+  is not an existing connection to the peer. In such a case, an
+  FI_EHOSTUNREACH should be expected.
```

Comment:
See above comment.  I would clarify that if multiple addresses are inserted at once, the flag applies to all of them.

It's also worth noting that the same error code may be returned if the target address becomes unreachable in the future